src/collective/geo/settings/upgrades/configure.zcml

```diff
@@ -0,0 +1,15 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    i18n_domain="collective.geo.settings">
+
+  <genericsetup:upgradeStep
+      title="Set new default value for plone 5"
+      source="31"
+      destination="32"
+      handler="collective.geo.settings.upgrades.migrate_default_marker_image.run"
```

Comment:
I would probably rename the module to something like `v23_ migrate_default_marker_image` in order to have a better naming system for when more upgrade steps are added.

Reply:
I thought about that too... thanks I change that.